### PR TITLE
bump deps

### PR DESCRIPTION
**Pull request by turadg**
## Description

- chore(deps): bump Prettier to 2.8.3
- chore(deps): remove unused "ses" dep
- chore(deps): yarn upgrade ava
- chore(deps): bump import-meta-resolve to 2.2.1

### Security Considerations

--

### Documentation Considerations

--
### Testing Considerations

CI should be enough. The only update bigger than .patch is import-meta-resolve which gets hit in CI.

### CODE REVIEW

File: packages/SwingSet/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "@endo/promise-kit": "^0.2.52",
     "@endo/zip": "^0.2.28",
     "anylogger": "^0.21.0",
-    "import-meta-resolve": "^1.1.1",
+    "import-meta-resolve": "^2.2.1",
```
Comment (dckc) [inline packages/SwingSet/package.json:52]: what prompted this change?

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "lerna": "^3.22.1",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.8.1",
+    "prettier": "^2.8.3",
```
Comment (dckc) [inline package.json:31]: Did anything in particular prompt this change?

File: yarn.lock [hunk 3]
```diff
@@ -4009,9 +4009,9 @@ atob@^2.1.2:
   integrity sha512-Wm6ukoaOGJi/73p/cl2GvLjTI5JM1k/O14isD73YML8StrH/7/lRFgmg8nICZgD3bZZvjwCGxtMOD3wWNAu8cg==
 
 ava@^5.1.0:
-  version "5.1.0"
-  resolved "https://registry.yarnpkg.com/ava/-/ava-5.1.0.tgz#98c4aac86a427c9843d3af13f25552b71014bff7"
-  integrity sha512-e5VFrSQ0WBPyZJWRXVrO7RFOizFeNM0t2PORwrPvWtApgkORI6cvGnY3GX1G+lzpd0HjqNx5Jus22AhxVnUMNA==
+  version "5.1.1"
```
Comment (dckc) [inline yarn.lock:4012]: did anything in particular prompt this? It seems like a considerable amount of churn for reasons that aren't clear to me. The [release notes](https://github.com/avajs/ava/releases/tag/v5.1.1) show only:
> - Fix de-registration of shared workers to ensure AVA exits correctly

**Review by dckc — COMMENTED** [review]
The ava upgrade seems to bring in a lot of indirect dependency changes; I'm not confident that I know the trade-offs in taking this upgrade.

File: packages/SwingSet/package.json (hunk 1 above)
Comment (turadg) [reply packages/SwingSet/package.json:52]: reviewing `yarn outdated`

File: package.json (hunk 2 above)
Comment (turadg) [reply package.json:31]: reviewing `yarn outdated` and being eager adopting in formatting tool changes to reduce churn (assuming over time we have more code that's likely to be affected)

File: yarn.lock (hunk 3 above)
Comment (turadg) [reply yarn.lock:4012]: "Churn" connotes to me something that affects developers. I don't see how this change in the lockfile has any negative consequences.

>  I'm not confident that I know the trade-offs in taking this upgrade.

What are the negative consequences you're trying to balance with the positive of having the current published versions?

Comment (dckc) [reply yarn.lock:4012]: Changes in dependencies might introduce a bug.

**Review by kriskowal — APPROVED** [review]

**Comment by dckc** [comment]
For reference:

 - #6632

File: packages/SwingSet/package.json (hunk 1 above)
Comment (dckc) [reply packages/SwingSet/package.json:52]: TIL.

**Pull request merged**